packages/shared/components/inputs/PinInput.svelte

```diff
@@ -16,14 +16,14 @@
     export let error: string
     export let label: string
 
-    let inputs = new Array(PIN_LENGTH)
+    let inputs = new Array(PINCODE_LENGTH)
```

Comment:
Don't add redundant information to variable names. It gets confusing. PIN stands for Personal Identification Number and even wiki says that PIN Code is redundant. 
![image](https://user-images.githubusercontent.com/45079109/194564593-15f49b4e-826f-4179-8e67-16c2c0d04574.png)

Reply:
Ah didn't realize this. Will remove redundancy